### ISSUE TITLE
Remove duplicate properties

### ISSUE THREAD

**Issue by MariskaS** [css/csso#278]
Hello!
If the selector 2 duplicate rules with different properties, if minification is only recently (see the task #257). But I need to remain both rules, as one used to Сhrom and the second for IE11
What to do in this case?

**Comment by lahmatiy** [comment]
Can you provide an example?

**Comment by MariskaS** [comment]
I am use angular-material. This library can meet the selectors with duplicate rules.And of course not only in this library can this be.
http://take.ms/FzN4b
https://github.com/angular/bower-material/edit/master/angular-material.css

**Comment by lahmatiy** [comment]
Yep, in this case we need to keep both. CSSO tries to keep properties with different support. In this example CSSO left both (try it [here](http://css.github.io/csso/csso.html)):

```css
.example {
    color: red;
    color: rgba(255, 0, 0, .5);
}
```

But CSSO doesn't know about `start` and `end` special values for `text-align` and wrongly merge them. That's can be fixed when issue is known. Probably we should rename issue to `text-align: start/end problem`...

**Comment by lahmatiy** [comment]
Problem with `text-align` is fixed in #281 
Closing issue for now, if any similar problem will be found – please, create a separate issue.

**Comment by MariskaS** [comment]
Thank you!
